__tests__/extension-requests/extension-requests.test.js

```diff
@@ -278,9 +278,9 @@ describe('Tests the Extension Requests Screen', () => {
         });
       } else if (
         url ===
```

Comment:
can these be stored to constants?

Reply:
it can but I do not think it is necessary since the url is not being repeated many times only once below